graylog-storage-opensearch2/src/test/java/org/graylog/storage/opensearch2/testing/DatanodeInstance.java

```diff
@@ -118,6 +118,9 @@ public GenericContainer<?> buildContainer(String image, Network network) {
                 .withEnv("GRAYLOG_DATANODE_ROOT_PASSWORD_SHA2", "<root-pw-sha2>")
                 .withEnv("GRAYLOG_DATANODE_MONGODB_URI", "mongodb://mongodb:27017/graylog")
                 .withEnv("GRAYLOG_DATANODE_SINGLE_NODE_ONLY", "true")
+                .withEnv("GRAYLOG_DATANODE_CONFIG_LOCATION", "/usr/share/graylog/datanode/bin/config")
+                .withEnv("GRAYLOG_DATANODE_INSECURE_STARTUP", "true")
```

Comment:
do we want or need the insecure startup flag in a test that verifies secured setup?

Reply:
I think we should leave it in for now